### PR TITLE
5.0.0.CR1 draft

### PR DESCRIPTION
**Pull request by vietj**

### CODE REVIEW

File: blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+
+Vert.x gRPC has a few great additions in this major release, propelling Vert.x gRPC as a first class choice for implementing
+gRPC services.
+
+### gRPC Web
+
+The Vert.x gRPC Server supports the [gRPC Web](https://github.com/grpc/grpc/blob/master/doc/PROTOCOL-WEB.md#protocol-differences-vs-grpc-over-http2)
+protocol by default (i.e. without an intermediate proxy).
+
+Since a full setup is quite complex, we will provide a how-to for this in the near future.
```
Comment (tsegismont) [inline blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]:
```suggestion
On the Vert.x side, enabling the gRPC Web protocol support is a mere switch in options.
In the near feature, we will provide a [tutorial](https://how-to.vertx.io/) explaining how to generate a browser client with `protoc` and to communicate with the server.
```

File: blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+Here you can find simple [examples](https://github.com/vert-x3/vertx-examples/tree/5.x/openapi-examples) for Vert.x OpenAPI.
+
+Here you can find simple [examples](https://github.com/vert-x3/vertx-examples/tree/5.x/web-examples/src/main/java/io/vertx/example/web/openapi_router) for Vert.x OpenAPI Router.
+
+## io_uring support
+
+Vert. 4 has been supporting [io_uring](https://en.wikipedia.org/wiki/Io_uring) in incubation for a while with a separate jar.
+
+Netty 4.2 has promoted io_uring out of incubation, as consequence Vert.x Core supports it out of the box.
```
Comment (tsegismont) [inline blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]:
```suggestion
Netty 4.2 has promoted io_uring out of incubation, as a consequence Vert.x Core supports it out of the box.
```

File: blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+```
+
+NOTE: `AbstractVerticle` is still fine and not deprecated, we simply recommend to use the base class that best works for 
+you
+
+## Java Modules
+
+Vert.x 5 supports the Java Platform Module System (JPMS) with explicit modules (Vert.x 4 supports only automatic modules).
+
+Most Vert.x components support modules (a few components though due to their dependencies cannot), the Modular Vert.x guide will give more information
```
Comment (tsegismont) [inline blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]:
```suggestion
Most Vert.x components support modules (a few components though due to their dependencies cannot), the upcoming Modular Vert.x guide will give more information
```

File: blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+Vert. 4 has been supporting [io_uring](https://en.wikipedia.org/wiki/Io_uring) in incubation for a while with a separate jar.
+
+Netty 4.2 has promoted io_uring out of incubation, as consequence Vert.x Core supports it out of the box.
+
+## Client side load balancing
+
+Client side load balancing for HTTP based clients, with a wide variety of policies.
+
+Here is a simple HTTP client [example](https://github.com/vert-x3/vertx-examples/blob/5.x/core-examples/README.adoc#load-balancing)
+as well as a gRPC HTTP client [example](https://github.com/vert-x3/vertx-examples/tree/5.x/grpc-examples#load-balancing).l
```
Comment (tsegismont) [inline blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]:
```suggestion
as well as a gRPC HTTP client [example](https://github.com/vert-x3/vertx-examples/tree/5.x/grpc-examples#load-balancing).
```

File: blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+
+- Vert.x Sync, replaced by Vert.x virtual threads 
+- Service Factories / Maven Service Factory / HTTP Service Factory 
+
+## Finally
+
+The [deprecations and breaking changes](https://github.com/vert-x3/wiki/wiki/5.0.0-Deprecations-and-breaking-changes)
+ can be found on the wiki.
+
+For this release there are no Docker images.,
```
Comment (tsegismont) [inline blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]:
```suggestion
For this release there are no Docker images.
```

Perhaps we should move this up to the sunset-ted components list.

**Review by tsegismont — COMMENTED** [review]
Great @vietj !

File: blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx (hunk 5 above)
Comment (vietj) [reply blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]: yes

Comment (vietj) [reply blog/2024-11-18-eclipse-vert-x-5-candidate-1-released.mdx]: will do, it's left over from the past

**Pull request merged**